setup.cfg

```diff
@@ -9,9 +9,14 @@ markers =
 
 [pylint]
 reports = no
-disable = locally-disabled,star-args,consider-using-f-string
+disable = line-too-long,locally-disabled,star-args,consider-using-f-string
```

Comment:
How is this related? I really do not like too long lines.

Reply:
I took this from the configuration of another of my Python package. But after re-reading the manual, it seems that it could be simplified. Interesting read: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html